db_interface/src/axolotl/IdentityKey.cpp

```diff
@@ -24,26 +25,29 @@ CriptextDB::IdentityKey CriptextDB::getIdentityKey(database db, string recipient
   return identityKey;
 }
 
-bool CriptextDB::createIdentityKey(database db, string recipientId, int deviceId, char *identityKey) {
+bool CriptextDB::createIdentityKey(database db, int accountId, string recipientId, int deviceId, char *identityKey) {
   try {
     bool hasRow = false;
     db << "begin;";
-    db << "Select * from identitykeyrecord where recipientId == ? and deviceId == ?;"
+    db << "Select * from identitykeyrecord recipientId == ? and deviceId == ? and accountId == ?;"
```

Comment:
`db << "Select * from identitykeyrecord where recipientId == ? and deviceId == ? and accountId == ?;"`